Makefile

```diff
@@ -0,0 +1,20 @@
+serve:
+	@ cd docs/html && if python -V 2>&1 | grep 'Python 2' > /dev/null; then \
+		python -m SimpleHTTPServer ;\
+	else \
+		python -m http.server ;\
+	fi
+
+clean:
+	find . -name build_out|xargs rm -rf
+	find . -name __pycache__|xargs rm -rf
+	find . -name "*.pyc"|xargs rm -rf
+	find . -name "*.swp"|xargs rm -rf
+	rm -rf docs/html
```

Comment:
Docs are no longer _really_ hosted in this repository, documentation-related targets should reside in the Makefile in the documentation repository.

Reply:
Ah, that'll explain why I couldn't find how to regenerate them. Is the best thing to do just to remove this line for now?

Reply:
Yes